docs/fundamentals/code-analysis/style-rules/ide0130.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "IDE0130: Namespace does not match folder structure"
+description: "Learn about code analysis rule IDE0130: Namespace does not match folder structure"
+ms.date: 12/05/2022
+ms.topic: reference
+f1_keywords:
+- IDE0130
+helpviewer_keywords:
+- IDE0130
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# Namespace does not match folder structure (IDE0130)
```

Comment:
IIRC, there was some requirement about a specific MSBuild property being set or something like that, can't recall exactly.

If that is still true, can you document it?

Reply:
It's only _sort_ of true. That's where the nuance in running with a workspace vs not comes in. To get in command line builds you have to set the project root, but for VS it is provided through the workspace context (via project system). 

I wasn't sure how much nuance we wanted to add to this page

Reply:
Running analyzers in command-line builds is a core scenario IMO.

If the user should do something special, it's better to be documented.

Reply:
Yes, I agree with @Youssef1313 .